boom/bootloader.py

```
@@ -878,8 +878,8 @@ def load_entries(machine_id: Optional[str] = None):
         entry_path = path_join(entries_path, entry)
         try:
             _add_entry(BootEntry(entry_file=entry_path))
-        except Exception as e:
-            _log_info("Could not load BootEntry '%s': %s", entry_path, e)
+        except Exception as err:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)